clouddriver-kubernetes-v2/src/main/java/com/netflix/spinnaker/clouddriver/kubernetes/v2/security/KubernetesV2Credentials.java

```diff
@@ -457,6 +457,14 @@ public String jobLogs(String namespace, String jobName) {
         "logs", KubernetesKind.JOB, namespace, () -> jobExecutor.jobLogs(this, namespace, jobName));
   }
 
+  public String jobLogs(String namespace, String jobName, String containerName) {
```

Comment:
It looks like we can delete the other `jobLogs` which is now only used it a test, which would allow us to also delete the other `KubectlJobExecutor.jobLogs()`.

Reply:
Yep, fooled by IntelliJ not graying the other one out. Removed!